File: DuDoong-Domain/src/main/java/band/gosrock/domain/domains/event/exception/EventNotOpenException.java
package band.gosrock.domain.domains.event.exception;


import band.gosrock.common.exception.DuDoongCodeException;

public class EventNotOpenException extends DuDoongCodeException {

    public static final DuDoongCodeException EXCEPTION = new EventNotOpenException();

    private EventNotOpenException() {
        super(EventErrorCode.EVENT_NOT_OPEN);
    }
}

File: DuDoong-Domain/src/main/kotlin/band/gosrock/domain/domains/event/exception/EventNotOpenException.kt
package band.gosrock.domain.domains.event.exception

import band.gosrock.common.exception.DuDoongCodeException

class EventNotOpenException private constructor() : DuDoongCodeException(EventErrorCode.EVENT_NOT_OPEN) {
    companion object {
        @JvmField
        val EXCEPTION: DuDoongCodeException = EventNotOpenException()
    }
}
